app-misc/double-commander/double-commander-1.0.9.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="Cross Platform file manager."
+HOMEPAGE="http://doublecmd.sourceforge.net/"
+SRC_URI="https://github.com/doublecmd/doublecmd/releases/download/v${PV}/doublecmd-${PV}-src.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+ABBREV="doublecmd"
+
+RESTRICT="strip"
+
+DEPEND="
+	>=dev-qt/qtcore-5.6
+	>=dev-qt/qtgui-5.6
+	>=dev-qt/qtnetwork-5.6
+	>=dev-qt/qtx11extras-5.6
+	dev-libs/libqt5pas:0/2.2"
+BDEPEND="
+	dev-lang/lazarus:0/2.2
+	net-misc/rsync"
+RDEPEND="
+	${DEPEND}
+	sys-apps/dbus
+	dev-libs/glib
+	x11-libs/libX11"
+
+S="${WORKDIR}/${ABBREV}-${PV}"
+
+src_prepare(){
+	eapply_user
+	use amd64 && export CPU_TARGET="x86_64" || export CPU_TARGET="i386"
+	export lazpath="/usr/share/lazarus"
+	find ./ -type f -name "build.sh" -exec sed -i 's#$lazbuild #$lazbuild --lazarusdir=/usr/share/lazarus #g' {} \;
```

Comment:
Missing || die, but this also doesn't respect prefix, so put ${EPREFIX} before /usr.